system-variables.md

```diff
@@ -957,6 +957,60 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 
 </CustomContent>
 
+### tidb_analyze_skip_column_types <span class="version-mark">New in v7.2.0</span>
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Default value: "json,blob,mediumblob,longblob"
+- Possible values: "json,blob,mediumblob,longblob,text,mediumtext,longtext"
+- This variable controls which types of columns are skipped for statistics collection when executing the `ANALYZE` command to collect statistics. The variable is only applicable for `tidb_analyze_version = 2`. When using the syntax of `analyze table t columns c1, ..., cn`, if the type of a specified column is included in `tidb_analyze_skip_column_types`, the statistics of this column will not be collected.
```

Comment:
```suggestion
- This variable controls which types of columns are skipped for statistics collection when executing the `ANALYZE` command to collect statistics. The variable is only applicable for `tidb_analyze_version = 2`. Even with the syntax  `analyze table t columns c1, ..., cn` to specify columns explicitly, the statistics of this column will not be collected either, in case the type of a specified column is included in `tidb_analyze_skip_column_types`.
```